.github/workflows/test_library_evolution.yml

```diff
@@ -28,9 +28,12 @@ jobs:
     - name: "xcodebuild clean archive BUILD_LIBRARY_FOR_DISTRIBUTION=YES"
       run: |
         xcodebuild -version
-        xcodebuild clean archive -project "${project}" -scheme "${scheme}" ${params} -destination "${destination}" | xcpretty --utf --color && exit ${PIPESTATUS[0]}
+        xcodebuild clean archive -project "${project}" -scheme "${scheme}" -destination 'generic/platform=iOS Simulator' -archivePath "${archivePath}" -skipPackagePluginValidation SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES
+        ./Scripts/validate-library-evolution.swift "${archivePath}" "${validationProjectName}"
+        cd ${validationProjectName}
+        xcodebuild clean build -scheme "${validationProjectName}" -destination 'generic/platform=iOS Simulator' | xcpretty --utf --color && exit ${PIPESTATUS[0]}
```

Comment:
Minor suggestion:
We can split those in 2 or 3 separate steps - Xcode build logs usually huge, so when if fail - we know where to look